main.go

```diff
@@ -315,6 +317,9 @@ func main() {
 	viper.SetDefault(enableFsGroupChangePolicyWebhookFlg, true)
 	enableFsGroupChangePolicyWebhook = viper.GetBool(enableFsGroupChangePolicyWebhookFlg)
 
+	viper.SetDefault(enableWalGEncryptionFlg, true)
```

Comment:
copy/paste error? or intentional?

```suggestion
	viper.SetDefault(enableWalGExporterFlg, true)
```